### DISCUSSION TITLE
Publish to GooglePlay and/or F-Droid?

### DISCUSSION THREAD

**Discussion by carcinocron** [Alex313031/Thorium-Android#5] [Ideas]
It would be nice to be able to install this via F-Droid. I couldn't find the browser in either store.

**Comment by ayoahha** [comment]
There is an issue on this problematic.
https://github.com/Alex313031/Thorium-Android/issues/11

But it is better i think to stay on the Discussion part to see if we can help authors to deploy this browser other than G***Play Store (if they aim to, hope not but...) !

I put here the last comment from this thread :
> I'd prefer if it was added to a repository that's not fdroid because fdroid requires all packages be built by the fdroid team and this could lead to the app being behind on updates on fdroid

@Ryhon0: I'm not sure that it exists for fdroid other type of repos.
There is the Obtainium solution, but this is not mature enough...

**Reply by ghost** [reply]
There's IzzyOnDroid repo. You can check it but I think a browser would be above thier apk size limit.
Bromite used to have their own repo.
Also would you consider adding thorium to privacy guides list? 
Users who use Foss and care about thier privacy really need a browser like this after bromite death.
Currently there's only brave which is full of crypto junk and unnecessary stuff.
I think thorium will be very appropriated by these communities. You just need it to be more user friendly to install and update

**Comment by Universalizer** [comment]
https://github.com/uazo/bromite-buildtools/issues/109

**Comment by gz83** [comment]
I don't think thorium can do that yet.
